security/production-secure-deploy-auto-gen-certs/README.rst

```diff
@@ -0,0 +1,402 @@
+Production recommended secure setup
+===================================
+
+Confluent recommends these security mechanisms for a production deployment:
+
+- Enable Kafka client authentication. Choose one of:
+
+  - SASL/Plain or mTLS
+
+  - For SASL/Plain, the identity can come from LDAP server
+
+- Enable Confluent Role Based Access Control for authorization, with user/group identity coming from LDAP server
+
+- Enable TLS for network encryption - both internal (between CP components) and external (Clients to CP components)
+
+In this deployment scenario, we will set this up, choosing SASL/Plain for authentication, using user provided CA certificate, CFK will auto generate the components certificates for you.
```

Comment:
Suggest changing this text to:

In this deployment scenario, we'll choose SASL/Plain for authentication and configure TLS encryption using CFK auto-generated component certificates. 
You'll need to provide a certificate authority certificate for CFK to auto-generate the component certificates.